aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
This doesn't make any sense to me. A single AimResource maps to a single ACI object, and therefore a single DN. BridgeDomain maps to fvBD, that's it. fvRsCtx is a child "object" that hangs off fvBD and doesn't identify the ACI object that BridgeDomain corresponds to. I wouldn't treat relationship MOs like fvRsCtx the same as MOs like fvBD which represent independent, concrete concepts. fvRsCtx is derived information that you can retrieve from the fvBD.
As far as the end user (in Neutron) is concerned, there is no value to showing the DN of the fvRsCtx object.